src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar-list calendar-list--start-day--wed calendar-list--month-length--29">
+      <li class="calendar-list__item "></li>
```

Comment:
you can simplify naming a little
```suggestion
    <ul class="calendar calendar--start-day--wed calendar--month-length--29">
      <li class="calendar__item "></li>
```